packages/ffe-cards-react/src/index.d.ts

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+export interface CardBaseProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
It's not a given that the resulting element is a div, so you should inherit React.HTMLAttributes<HTMLElement> instead.